Sources/Crypto/Util/SecureBytes.swift

```diff
@@ -327,6 +330,15 @@ extension SecureBytes {
             }
         }
 
+        @usableFromInline
+	var allocatedCapacity: Int {
```

Comment:
```suggestion
        var allocatedCapacity: Int {
```

Reply:
I think I missed a push here which fixed this formatting error. This should be fixed now.